s4e-web/src/app/views/map-view/view-manager/overlay-list-modal/overlay-list-modal.component.html

```diff
@@ -13,6 +13,54 @@
       </div>
 
       <ng-container *ngIf="showOverlayForm$ | async">
+        <div>
+          <p>Poprawny link zdjęcia WMS powinien zawierać:</p>
+          <ol>
+            <li>
+              <b>VERSION=</b>
+              wersja zapytania w formacie <b>x.y.z</b>
+            </li>
+            <li>
+              <b>REQUEST=</b>
+              typ zapytania, akceptowane wartości <b>GetMap</b>
+            </li>
+            <li>
+              <b>LAYERS=</b>
+              lista oddzielonych przecinakmi nazw warstw lub pojedyncza nazwa
+            </li>
+            <li>
+              <b>STYLES=</b>
+              lista oddzielonych przecinkami typów renderignu dla każdej
+              z podanych warstw lub pojedyncza nazwa
+            </li>
+            <li>
+              <b>SRS=</b> lub <b>CRS=</b>
+              typ projekcji zdjęcia WMS w formacie <b>nazwa przestrzeni:identyfikator</b>
+            </li>
+            <li>
+              <b>BBOX=</b>
+              współrzędne wierzchołków przestrzeni <b>minx,miny,maxx,maxy</b>
+              (dolny lewy, górny prawy) w jednostkach SRS
+            </li>
+            <li>
+              <b>WIDTH=</b>
+              szerokość pobieranego zdjęcia w pixelach
+            </li>
+            <li>
+              <b>HEIGHT=</b>
+              wysokość pobieranego zdjęcia w pixelach
+            </li>
+            <li>
+              <b>FORMAT=</b>
+              format zdjęcia dozwolone <b>vnd.jpeg-png|vnd.jpeg-png8|png|gif|tiff|jpg</b>
+            </li>
```

Comment:
I don't think user should specify these params, in fact, specifying them will be superfluous. At least BBOX, WIDTH and HEIGHT will surely be set by openlayers dynamically (they depend on browser visible map portion). FORMAT probably also be set by OL, and it will expect png output most likely, although this would need verification.

I started reading below before proceeding with this comment and I see that you remove a bunch of params anyway (including the ones I mention). If you strip them, why allow them in the first place? Wouldn't it be more obvious for the user, if she only could specify LAYERS and STYLES? Others are ignored anyway, which will be confusing. I would only allow these two, and reject any other specified.

Reply:
In moment I will add `(optional)` string to this description

Reply:
Good idea with leaving only LAYERS and STYLES

Reply:
Add a note to each, that it's value is disregarded as well